project/plugins.sbt

```diff
@@ -1,10 +1,4 @@
 addSbtPlugin(dependency = "ch.epfl.scala" % "sbt-version-policy" % "3.1.0")
-addSbtPlugin(dependency = "com.github.sbt" % "sbt-release" % "1.1.0")
 addSbtPlugin(dependency = "com.github.sbt" % "sbt-unidoc" % "0.5.0")
-addSbtPlugin(dependency = "com.github.sbt" % "sbt-pgp" % "2.1.2")
-addSbtPlugin(dependency = "com.timushev.sbt" % "sbt-updates" % "0.6.4")
 addSbtPlugin(dependency = "com.typesafe.play" % "sbt-plugin" % "2.9.0")
-addSbtPlugin(dependency = "net.vonbuchholtz" % "sbt-dependency-check" % "5.1.0")
-addSbtPlugin(dependency = "org.scoverage" % "sbt-scoverage" % "2.0.9")
-addSbtPlugin(dependency = "org.scoverage" % "sbt-coveralls" % "1.3.11")
```

Comment:
I removed the coverage thing, not sure if someone even pays attention to that.
Also I think if we introduce something like that we might want to have something we agree on for all the repos.